roles/kubernetes-apps/cephfs_provisioner/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+cephfs_provisioner_image_repo: quay.io/kubespray/cephfs-provisioner
+cephfs_provisioner_image_tag: latest
```

Comment:
can we tag with the version ?

Reply:
Yes for sure, but I have no idea about how to name it... upstream last tag was v0.1.2 only but half year ago...

v0.1.3? Or v1.0.0? Or v1.0.0-alpha1?

I don’t hope to make user confuse when upstream get a new stable tag... so simply keep it as latest now...

Reply:
the gitsha ?

Reply:
Oh good idea @_@!! Let's do so~~~